testsuite/tests/kuadrant/authorino/authorization/opa/conftest.py

```diff
@@ -0,0 +1,27 @@
+"""Conftest for Open Policy Agent (OPA)"""
+import pytest
+
+from testsuite.openshift.mockserver import Mockserver
+
+
+@pytest.fixture(scope="module")
+def header():
+    """Header used by OPA policy"""
+    return "opa", "opa-test"
+
+
+@pytest.fixture(scope="module")
+def rego_query():
+    """Rego query that allows all requests that contain `key` and `value`"""
+    def _rego(key, value):
+        return f"allow {{ input.context.request.http.headers.{key} == \"{value}\" }}"
```

Comment:
I know it is code repetition but I would probably use this explicitly instead so its more explicit when reading the test, or alternatively move this to utils and give it a better name